apps/mobile/src/navigation/TabsNavigator.js

```diff
@@ -52,6 +57,20 @@ export default function TabsNavigator () {
     <Tabs.Navigator {...navigatorProps}>
       <Tabs.Screen name='Home Tab' component={HomeNavigator} />
       <Tabs.Screen name='Messages Tab' component={MessagesNavigator} options={{ tabBarBadge: messagesBadgeCount }} />
+      <Tabs.Screen
+        name='Notifications Tab'
+        component={HomeNavigator} // it will never navigate to this but we need to pass a valid component here anyway
```

Comment:
You can just `() => {}` this (or create above outside the TabsNavigator component a DummyComponent with that definition)  and remove the comment. I think that would be clear enough and also not ever risk something rendering here that shouldn't . if it breaks it would be more clear.